design/media_types.go

```diff
@@ -90,15 +90,18 @@ var workItemType = MediaType("application/vnd.workitemtype+json", func() {
 	Attribute("version", Integer, "Version for optimistic concurrency control")
 	Attribute("name", String, "User Readable Name of this item type")
 	Attribute("fields", HashOf(String, fieldDefinition), "Definitions of fields in this work item type")
+	Attribute("parentPath", String, "The names of the parent types separated with '/' or just '/' if no parents")
 
 	Required("version")
 	Required("name")
 	Required("fields")
+	Required("parentPath")
 
 	View("default", func() {
 		Attribute("version")
 		Attribute("name")
 		Attribute("fields")
+		Attribute("parentPath")
```

Comment:
I would not expose the parent path to clients until we have a need for it. So far inheritance is only know to clients when creating work item types, not when using them, so this would be a larger design change than you would think.